index.html

```diff
@@ -1,20 +1,39 @@
 <!DOCTYPE html>
-<html lang="pl">
-<head>
-    <meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
-    <title>CodersCamp2020 | Star Wars API QUIZ</title>
-    <link rel="stylesheet" href="styles/App.css"/>
+<html lang="en">
 
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-          rel="stylesheet">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CodersCamp2021 | Harry potter API QUIZ</title>
+    <link rel="stylesheet" href="./styles/index.css">
 </head>
+
 <body>
+    <div class="game-wrapper">
+        <header class="header-content">
+            <div class="header-content__game-logo"></div>
+            <div class="header-content__game-nav"> </div>
+        </header>
 
-<div id="swquiz-app">
-</div>
+        <aside class="game-image">
 
-<script src="src/index.js"></script>
+        </aside>
+
+        <main class="game">
+            <div class="game__mode"></div>
+            <div class="game__mode-rules"></div>
+            <div class="game__btns">
+                <button class="game__ranking game__rules"></button>
+                <button class="game__play"></button>
```

Comment:
We could consider adding #ids to the buttons, we will probably need to get them from js side. We can use querySelector also thuogh, but we can cosider ids.

Reply:
Ok, it's not a problem :)